app/controllers/articles_controller.rb

```diff
@@ -72,6 +73,10 @@ def eds_init
       field.label = 'Abstract'
     end
 
+    config.add_search_field('series') do |field|
+      field.label = 'Series'
```

Comment:
Examples article search: http://localhost:3000/articles?utf8=%E2%9C%93&search_field=search&q=SE+%22SpringerBriefs+in+Environmental+Science%22